include/merlin_hashtable.cuh

```diff
@@ -258,15 +258,15 @@ class HashTable {
       check_evict_strategy(metas);
     }
 
+    std::shared_lock<std::shared_timed_mutex> lock(mutex_, std::defer_lock);
+    if (!reach_max_capacity_) {
+      lock.lock();
+    }
+
     if (is_fast_mode()) {
-      const size_t block_size = 128;
+      constexpr size_t block_size = 128;
```

Comment:
Wow, it's an existing bug, pls change it to option_.block_size, could you add new test case for none 128 block_size?

Reply:
Ok, that answers the question I had in Slack. Should I also replace all other instances where you do not use the `options_.block_size`?

Reply:
Done as discussed on Slack!